react-backend/routes/qr_generator.js

```diff
@@ -24,7 +29,8 @@ const schema = {
   },
 };
 
-router.post('/', validate(schema), (req, res, next) => {
+router.post('/', validate(schema), async (req, res, next) => {
+  console.log('this is post');
```

Comment:
Oops, stray console.log

Reply:
Oopsieee, should be removed now. Regarding the browser console errors, I was just gonna ignore them for now since they're just warnings and raise an issue; so I can get these pr's done.

If that's not cool, I'm happy to sort it now 🤷‍♂️